File: BackEnd/RestaurantApiProject/Models/Food.cs
using RestaurantApiProject.Contracts;
using System;
using System.Collections.Generic;

// Code scaffolded by EF Core assumes nullable reference types (NRTs) are not used or disabled.
// If you have enabled NRTs for your project, then un-comment the following line:
// #nullable disable

namespace RestaurantApiProject.Models
{
    public partial class Food: IEntity
    {
        public Food()
        {
            OrdersFood = new HashSet<OrdersFood>();
        }

        public int Id { get; set; }
        public string FoodType { get; set; }
        public string FoodName { get; set; }
        public decimal FoodPrice { get; set; }
        public string FoodDescription { get; set; }
        public DateTime CreateDate { get; set; }

        public virtual ICollection<OrdersFood> OrdersFood { get; set; }
    }
}

File: Baladina_Mobile_App/baladina_mobile_app/lib/data/Models/food.dart
class Food {
  late int id;
  late String foodType;
  late String foodName;
  late double foodPrice;
  late String foodDescription;
  late String createDate;

  Food.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    foodType = json['foodType'];
    foodName = json['foodName'];
    foodPrice = json['foodPrice'];
    foodDescription = json['foodDescription'];
    createDate = json['createDate'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['foodType'] = this.foodType;
    data['foodName'] = this.foodName;
    data['foodPrice'] = this.foodPrice;
    data['foodDescription'] = this.foodDescription;
    return data;
  }
}
